### PR TITLE
Normalize directory names in index created by dir2pi to lower case (fixe...

### PR DESCRIPTION
**Pull request by orodbhen**
...s Issue #34).

### CODE REVIEW

**Comment by wolever** [comment]
Hey! First, I'm very sorry about the delay in responding to this issue. Please forgive me!

Second: I'm a bit wary of making this change as it may cause surprises for existing users on case-sensitive filesystems. Could you tell me a bit more about your motivation for normalizing the case in directory names?

**Comment by wolever** [comment]
Oh! I'd missed https://github.com/pypa/pip/issues/2136 — let me look into this, but if this is the behavor recommended by dstufft then I'll get this put in.

**Comment by wolever** [comment]
Okay, closing out this PR; discussion will continue at https://github.com/wolever/pip2pi/issues/37

File: libpip2pi/commands.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ def dir2pi(argv=sys.argv, use_symlink=OS_HAS_SYMLINK):
         if pkg_basename.startswith("."):
             continue
         pkg_name, pkg_rest = file_to_package(pkg_basename, pkgdir)
-        pkg_dir = pkgdirpath("simple", pkg_name)
+        pkg_dir = pkgdirpath("simple", pkg_name.lower())
```
Comment (jchysk) [inline libpip2pi/commands.py]: Maybe there should be a replace('_', '-') to handle underscores to hyphens in addition to lower() for normalizing the directory names. Packages that have underscores in them still receive the deprecation warning.

**Pull request closed without merging**